packages/nft-base-transactions/src/handlers/nft-register-collection.ts

```diff
@@ -11,15 +11,21 @@ import { INFTCollections } from "../interfaces";
 import { NFTIndexers } from "../wallet-indexes";
 import { NFTBaseTransactionHandler } from "./nft-base-handler";
 
+const pluginName = require("../../package.json").name;
+
 @Container.injectable()
 export class NFTRegisterCollectionHandler extends NFTBaseTransactionHandler {
     @Container.inject(Container.Identifiers.PluginConfiguration)
-    @Container.tagged("plugin", "@protokol/nft-base-transactions")
+    @Container.tagged("plugin", pluginName)
     private readonly configuration!: Providers.PluginConfiguration;
 
     @Container.inject(Container.Identifiers.TransactionHistoryService)
     private readonly transactionHistoryService!: Contracts.Shared.TransactionHistoryService;
 
+    @Container.inject(Container.Identifiers.CacheService)
+    @Container.tagged("cache", pluginName)
+    private readonly tokenSchemaValidator!: Contracts.Kernel.CacheStore<string, any>;
```

Comment:
`tokenSchemaValidator` -> `tokenSchemaValidatorCache` 

so you know later on what kind of object/variable you are working with. a small nitpick